src/containers/preview-modal.jsx

```diff
@@ -0,0 +1,68 @@
+import bindAll from 'lodash.bindall';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+
+import PreviewModalComponent from '../components/preview-modal/preview-modal.jsx';
+import BrowserModalComponent from '../components/browser-modal/browser-modal.jsx';
+
+import {
+    closePreviewInfo
+} from '../reducers/modals';
+
+class PreviewModal extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleTryIt',
+            'handleCancel'
+        ]);
+
+        this.state = {
+            previewing: false
+        };
+    }
+    handleTryIt () {
+        this.setState({previewing: true});
+        this.props.onTryIt();
+    }
+    handleCancel () {
+        window.history.back();
+    }
+    supportedBrowser () {
+        const userAgent = (typeof navigator === 'undefined') ? '' : navigator.userAgent;
+        if (userAgent === '' || userAgent.indexOf('Chrome') > 0 || userAgent.indexOf('Trident') > 0) {
```

Comment:
Probably not necessary, but wondering if we should use a library for this? Just in the case that the library has a maintainer that is keeping more up to date with any `userAgent` changes that might occur.

Reply:
Agreed. This can often be notoriously tricky. I can recommend both of these two options:
https://www.npmjs.com/package/bowser
https://www.npmjs.com/package/platform